config/routes.rb

```diff
@@ -6,8 +6,11 @@
   root 'start#index'
 
   get 'cookies' => 'start#cookies'
-  get 'select_soc_codes' => 'start#select_soc_codes'
-  get 'report' => 'start#report'
+  post 'keywords' => 'start#keywords'
+  get ':guid/select_soc_codes' => 'report#select_soc_codes'
+  post ':guid/save_soc_codes' => 'report#save_soc_codes'
```

Comment:
oh, they're action method. isn't life going to be easier if we model this with resources?